### ISSUE TITLE
Document Attribute Tags with Render Body - Render Body Must Come After Attribute Tags

### ISSUE THREAD

**Issue by austingardner** [marko-js/marko#1678]
### Details

The following code will produce an error: 
```
<ebay-snackbar-dialog open=state.open on-close('handleClose') on-action("handleAction")> 
    Action Snackbar 
    <@action access-key="Undo">
      Undo
    </@action> 
</ebay-snackbar-dialog> 
```
But this code doesn't:

```
<ebay-snackbar-dialog open=state.open on-close('handleClose') on-action("handleAction")> 
    <@action access-key="Undo">
      Undo
    </@action>
Action Snackbar
</ebay-snackbar-dialog> 
```

### Expected Behavior

We should document the fact that renderBody must come after attribute tags when both are present inside a custom tag. I couldn't find it anywhere in the attribute tags page in the [docs](https://markojs.com/docs/body-content/#passing-attributes-to-body-content)

### Stack Trace

Throws error `cannot set property start of undefined`

</details>

**Comment by tigt** [comment]
Looks like this no longer errors in Marko 6, at least: https://markojs.com/playground/#NobwRAdghgtgpmAXGAlhAJnAHgOhlAJwGsB7MAGjAAcoAXACyTAHoBjEmKkiOCWgZ2ZpMufMTKV2fXrSYAeADZQAniQCutAHwAdCAAI9AIzgAzEgTi6DcgAL04UdGgDmehXABucBQF4AzJoAEt4KJHoAsoSkcsx2Dk4Qzjr6elAmtHAEujFKqhqaYAC+5ODQ8Ey56rR4URLUdIzIbBxcPHyClRo14hRgUhl88vxwrLQo3MnWACQgAOT0s3oA1HpoVF32ji447l4KhZMGenIzaxvx2xYYmQBCJOjKhcyHMS+nEOvVV5gEdw9PyRiw1G4wgBUKAF0gA

**Comment by tigt** [comment]
Closing since this was apparently fixed for Marko 5 by #1355 and the above playground link shows that fix persists to 6

**Comment by DylanPiercey** [comment]
#1355 is not related to this issue and it still needs to be addressed.
